modules/virt-define-http-readiness-probe.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-define-http-readiness-probe_{context}"]
+
+= Defining an HTTP readiness probe
+
+Define an HTTP readiness probe by setting the `spec.readinessProbe.httpGet` field of the virtual machine instance (VMI) configuration.
+
+
+.Procedure
+. Include details of the readiness probe in the VMI configuration file.
++
+
+.Sample readiness probe with an HTTP GET test
+[source,yaml]
+----
+spec:
+  readinessProbe:
+    httpGet: <1>
+      port: 1500 <2>
```

Comment:
I think we should keep this similar to the one below and also show/explain the full example (including path).
So the examples are consistent.

Reply:
Added the path and httpHeaders fields similar to the liveness probe example